sdk/monitor/opentelemetry-exporter-azuremonitor/CHANGELOG.md

```diff
@@ -1,17 +1,18 @@
 # Release History
 
-## Unreleased
+## 1.0.0b1 (Unreleased)
```

Comment:
Are we changing the version?

Reply:
I believe this also doesn't follow semver? (https://github.com/open-telemetry/opentelemetry-python/issues/1222#issuecomment-705796935)

Reply:
We would release this package eventually as the first preview of the major version. 

I've just updated it with the language precedent set. Perhaps, we can discuss the version offline?

Reply:
also, we currently adhere to semvers for all the GAed libraries- and for pre releases, we are compatible with pep 440 , https://www.python.org/dev/peps/pep-0440/#id27